src/ovh-contacts.service.js

```diff
@@ -155,49 +149,42 @@ export default class OvhContactsService {
       })
       .then(rulesOptions => this.$q.all({
         apiSchemas: this.getMeSchemas(),
-        creationRules: this.OvhApiNewAccount.v6().rules({}, rulesOptions).$promise,
+        nicCreationRules: this.OvhApiNewAccount.v6().rules({}, rulesOptions).$promise,
```

Comment:
It would be better to have an action verb like `getRules` than just `rules` imo

Reply:
In fact... It's a post to the API :-)